doc/configuration.md

```diff
@@ -0,0 +1,206 @@
+                                   :ssl-keystore-location "/location/to/keystore"
+                                   :ssl-keystore-password "some-password"
+                                   {:jaas {:username "username"
+                                           :password "password"
+                                           :mechanism "SCRAM_SHA-512"}}}
+            :default-api-timeout-ms-config [600000 :int]
+            :statsd               {:host    "localhost"
+                                   :port    [8125 :int]
+                                   :enabled [false :bool]}
+            :statsd               {:host    "localhost"
+                                   :port    [8125 :int]
+                                   :enabled [false :bool]}
+            :sentry               {:enabled                   [false :bool]
+                                   :dsn                       "dummy"
+                                   :worker-count              [5 :int]
+                                   :queue-size                [5 :int]
+                                   :thread-termination-wait-s [1 :int]}
+            :rabbit-mq-connection {:host            "localhost"
+                                   :port            [5672 :int]
+                                   :prefetch-count  [3 :int]
+                                   :username        "guest"
+                                   :password        "guest"
+                                   :channel-timeout [2000 :int]}
+            :rabbit-mq            {:delay       {:queue-name           "application_name_delay_queue"
+                                                 :exchange-name        "application_name_delay_exchange"
+                                                 :dead-letter-exchange "application_name_instant_exchange"
+                                                 :queue-timeout-ms     [5000 :int]}
+                                   :instant     {:queue-name    "application_name_instant_queue"
+                                                 :exchange-name "application_name_instant_exchange"}
+                                   :dead-letter {:queue-name    "application_name_dead_letter_queue"
+                                                 :exchange-name "application_name_dead_letter_exchange"}}
+            ;; if retry is enabled, messages are retried in RMQ.  If retry is disabled, and :retry is returned from mapper function, messages will be lost.
+             :retry                {:count   [5 :int]
+                                   :enabled [false :bool]}
+            :jobs                 {:instant {:worker-count   [4 :int]
+                                             :prefetch-count [4 :int]}}
+            :http-server          {:port         [8010 :int]
+                                   :graceful-shutdown-timeout-ms [30000 :int]
+            :new-relic            {:report-errors [false :bool]}}}}
+```
+
+
+## General Configurations
+
```

Comment:
Also add column to mark required and optional field to minimize the spread of configmap

Reply:
done